src/app/app.module.ts

```diff
@@ -0,0 +1,35 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { ErrorHandler, NgModule } from '@angular/core';
+import { IonicApp, IonicErrorHandler, IonicModule } from 'ionic-angular';
+import { SplashScreen } from '@ionic-native/splash-screen';
+import { StatusBar } from '@ionic-native/status-bar';
+
+import { MyApp } from './app.component';
+import { HomePage } from '../pages/home/home';
+import {LoginPage} from "../pages/login/login";
+import { AuthServiceProvider } from '../providers/auth-service/auth-service';
```

Comment:
I recommend using comments to separate imports. 
For example:
//@pages
`import { HomePage } from '../pages/home/home';`
 `import { LoginPage } from "../pages/login/login";`
//@providers
`import { AuthServiceProvider } from '../providers/auth-service/auth-service';`